lib/oca-epak/epak/client.rb

```diff
@@ -14,23 +14,7 @@ def check_credentials
         method = :get_epack_user
         opts = { "usr" => username, "psw" => password }
         response = client.call(method, message: opts)
-        parse_results_table(response, method)[:existe] == "1"
-      end
-
-      # Checks whether the operation is valid
-      #
-      # @param [String] Client's CUIT
-      # @param [String] Operation Type
-      # @return [Boolean]
-      def check_operation(cuit, op)
```

Comment:
@etagwerker I always felt it was a poor implementation, it asked the Oca API for a shipping rate check using random details, because the `get_operation_codes` method was not available at the time. Since that method is now available, it doesn't make much sense to have that now. Another solution would be to re-implement `check_operation` using `get_operation_codes` by checking the operations in the resulting array, which would be much better but I don't see much use for it.